src/core/or/circuitmux_ewma.c

```diff
@@ -216,6 +216,9 @@ static int
 ewma_cmp_cmux(circuitmux_t *cmux_1, circuitmux_policy_data_t *pol_data_1,
               circuitmux_t *cmux_2, circuitmux_policy_data_t *pol_data_2);
 
+static void
+ewma_touch(circuitmux_policy_data_t *pol_data);
```

Comment:
We should not have a newline between the return type and the function name when we are declaring the function: only when we are defining it.